src/Utils/types.ts

```diff
@@ -0,0 +1,10 @@
+export type Login = {
```

Comment:
I think that creating huge file for every type is kinda bad

I suggest to create file /types/requestTypes (smth like that, come up with better name pls)

Reply:
Done